moshi/src/test/java/com/squareup/moshi/JsonReaderTest.java

```diff
@@ -297,7 +297,7 @@ JsonReader newReader(String json) throws IOException {
         + "2.718281828459045]";
     JsonReader reader = newReader(json);
     reader.beginArray();
-    assertThat(reader.nextDouble()).isEqualTo(-0.0);
+    assertThat(reader.nextDouble()).isIn(0.0, -0.0);
```

Comment:
The distinction ±0.0 is lost in the `JsonValueReader` since `BigDecimal` does not distinguish between those.